dockstore-webservice/src/main/java/io/dockstore/webservice/resources/NotificationResource.java

```diff
@@ -0,0 +1,110 @@
+package io.dockstore.webservice.resources;
+
+import java.util.List;
+
+import javax.annotation.security.RolesAllowed;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import io.dockstore.webservice.CustomWebApplicationException;
+import io.dockstore.webservice.core.Notification;
+import io.dockstore.webservice.jdbi.NotificationDAO;
+import io.dropwizard.hibernate.UnitOfWork;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.Authorization;
+import org.apache.http.HttpStatus;
+import org.hibernate.SessionFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static io.dockstore.webservice.Constants.JWT_SECURITY_DEFINITION_NAME;
+
+@Path("curation")
+@Api("curation")
+@Produces(MediaType.APPLICATION_JSON)
+public class NotificationResource {
+    private static final Logger LOG = LoggerFactory.getLogger(OrganizationResource.class);
+
+    // interface between the endpoint and the database
+    private final NotificationDAO notificationDAO;
+    private final SessionFactory sessionFactory;
```

Comment:
I don't think you need to save this; you're not using it anywhere that I can see